draft-ietf-wish-whip.md

```diff
@@ -185,6 +185,8 @@ A WHIP client sending a PATCH request for performing ICE restart MUST contain an
 
 If the HTTP PATCH request results in an ICE restart, the WHIP resource SHALL return a 200 OK with an "application/trickle-ice-sdpfrag" body containing the new ICE username fragment and password and, optionally, the new set of ICE candidates for the media server and the new entity-tag corresping to the new ICE session in an ETag response header.
 
+If the ICE request can not be performed by the WHIP resource it MUST return an appropiate HTTP error code but MUST NOT terminate the session inmediately. The WHIP client COULD try to agin to perfroma new ICE restart or terminate the session issuing a HTTP DELETE request instead. In any case the session MUST be terminated if the ICE consent expires.
+
```

Comment:
Small typos:

agin/again
perfroma/perform

In the last sentence, I'd say something like

> In any case, the session MUST be terminated if the ICE consent expires as a consequence of the failed ICE request

since it's in the context of failed requests.